### ISSUE TITLE
USER ISSUE: Changing cart owner

### ISSUE THREAD

**Issue by Gingerbeard4251** [StrangeLoopGames/EcoIssues#24179]
\
**User Id: (slg43475)**
**Version:** 0.10.0.0 beta release-532

**Steps to Reproduce:**
change cart deed owner

**Expected behavior:**

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Property.Deed.CanChangeOwner(Player changer, IAlias owner)
   at Eco.Game</size>...

Check log file to see full message.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
no mods

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by Gingerbeard4251** [comment]
Server log error

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at Eco.Shared.Networking.RPCManager.Invoke(INetClient client, RPCInvocation invocation)
   at Eco.Shared.Networking.RPCManager.InvokeOn(INetClient client, BSONObject bson, Object target, String methodName)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Plugins.Networking.NetworkServer.Eco.Shared.Networking.INetworkEventHandler.ReceiveEvent(INetClient client, NetworkEvent netEvent, BSONValue bsonValue)
   at Eco.Plugins.Networking.Client.<.ctor>b__78_4()
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at Eco.Shared.Networking.RPCManager.Invoke(INetClient client, RPCInvocation invocation)
   at Eco.Core.Controller.ControllerManager.HandleViewRPC(INetClient client, Int32 universalID, String methodIdOrName, BSONObject bson)
   at InvokeStub_Client.ViewRPC(Object, Object, IntPtr*)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Property.Deed.CanChangeOwner(Player changer, IAlias owner)
   at Eco.Gameplay.Property.Deed.TryChangeOwners(GameActionPack pack, Player changer, IAlias newOwner, Boolean transferAction, Boolean autoPerform, Boolean okBox, Boolean superAccess)
   at Eco.Gameplay.Property.Deed.ChangeOwners(Player changer, IAlias newOwner)
   at InvokeStub_Deed.ChangeOwners(Object, Object, IntPtr*)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)

**Comment by D3nnis3n** [comment]
Fixed in 10.0.1
